src/reduced_basis/rb_parametrized_function.C

```diff
@@ -31,6 +31,8 @@ namespace libMesh
 RBParametrizedFunction::RBParametrizedFunction()
 :
 requires_xyz_perturbations(false),
+is_lookup_table(false),
+lookup_table_param_name(""),
```

Comment:
An empty string is constructed by default so this one line isn't needed, but definitely do need to initialize the bool!

Reply:
It might not hurt to leave the line in (IIRC there are some ultraparanoid warning levels we aren't close to being able to turn on that want you to explicitly write a constructor for every member), but I wonder if the compiler has an easier time optimizing () (default constructor, clearly no data to copy) over ("") (const char * overload, but hopefully inlined so maybe it can figure out that in this particular case there's no data to copy?).

Reply:
Any compiler warning about not default-initializing a string (or any STL container) is definitely in the wrong IMO. I'm trying to imagine all the STL member variables we have and then imagine the compiler warning about every one of them that's not explicitly default-initialized...